tools/quickstart/quickstart/docker.py

```diff
@@ -19,10 +21,40 @@
 LEGACY_CONTAINER_NAMES = ["watchtower-testnet", "trustlines-testnet"]
 
 
-def update_and_start(host_base_dir: str) -> None:
-    if not os.path.isfile("docker-compose.yaml") and not os.path.isfile(
-        "docker-compose.yml"
-    ):
+def setup_interactivaly(base_dir, docker_compose_file):
+    if does_docker_compose_file_exist(base_dir):
+        while True:
```

Comment:
How about we check if the two files are equal first (e.g. with `filecmp.cmp`) and if so just do nothing? Then we wouldn't have to bother the user in the normal case that they didn't touch the docker file.

Reply:
I can do that, but we would still bother him in case we changed something, but he did still not touch it. I thought about having a list of old file hashes, but decided that it is not forth the work at the moment. 
I can add the same file check for now.